README.md

```diff
@@ -20,14 +20,15 @@ PyHa = Python + Piha (referring to a bird species of our interest known as the s
 
 1. Navigate to a desired folder and clone the repository onto your local machine. `git clone https://github.com/UCSD-E4E/PyHa.git`
 
-- If you wish to reduce the size of the repository on your local machine you can alternatively use `git clone https://github.com/UCSD-E4E/PyHa.git --depth 1` which will only install the most up-to-date version of the repo without its history.
+    - If you wish to reduce the size of the repository on your local machine you can alternatively use `git clone https://github.com/UCSD-E4E/PyHa.git --depth 1` which will only install the most up-to-date version of the repo without its history.
 
-2. Install Python 3.8, Python 3.9, or Python 3.10
+2. Install Python 3.8, Python 3.9, or Python 3.10. *Make sure you install the 64-bit version.*
+    - Install links can be found [here.](https://www.python.org/downloads/release/python-31011/)
```

Comment:
Note that the latest 3.10 release is 3.10.14, not 3.10.11.

For Ubuntu users, this is probably not the appropriate source for Python - should probably use the package from `apt` or `snap`.

This also doesn't help people who already have `anaconda` or `miniconda` installed.